pkg/collector/corechecks/snmp/config_test.go

```diff
@@ -105,6 +105,20 @@ tags:
 profiles:
   f5-big-ip:
     definition_file: f5-big-ip.yaml
+    inline-profile:
```

Comment:
can you have both `inline-profile` and `definition_file` ?

Reply:
@pducolin this `inline-profile` was unintentionally committed, just cleaned that. `inline-profile` here is not correctly indented. See correct syntax in TestInlineProfileConfiguration test.